ui.apps/src/main/content/jcr_root/apps/saas-aem-module/components/search/search.html

```diff
@@ -1,6 +1,6 @@
 <sly data-sly-use.model="com.valtech.aem.saas.api.fulltextsearch.SearchModel"
      data-sly-test.isEdit="${wcmmode.edit}"></sly>
-<sly data-sly-use.template="/libs/core/wcm/components/commons/v1/templates.html"
+<sly data-sly-use.template="core/wcm/components/commons/v1/templates.html"
```

Comment:
we may want to change this in case this is the only dependency to core components. adding our own template is no big deal